data/human/free worlds start.txt

```diff
@@ -1582,67 +2087,60 @@ mission "FW Pirates 4.1"
 				`	"I trusted that Tomek would know the best strategy."`
 				`	"I guess I got too caught up in trying to be a hero."`
 					goto hero
-			`	"And how did that work out?" says the old man. "How long after the battle did it take you to realize that we have no way of holding on to the planet you conquered?"`
-			`	"Not long," you admit.`
+			`	"And how did that work out?" says the old man. "Shortly after your attack on Greenrock, the Navy made moves to invade us once again, a plan that was no doubt helped by your actions."`
 				goto sentence
 			label hero
 			`	"Indeed," says the senator. "And I bet it wasn't long after the battle that you realized saving the galaxy isn't that easy."`
-			`	"I suppose so," you admit.`
 			label sentence
+			`	"I suppose so," you admit.`
 			`	A third senator speaks up, a woman whose name card identifies her as Sara, from Skymoot. "That will be all, Captain <last>. The Senate will confer and decide whether we want your continued services."`
-			`	You are led back out to the lobby, where you wait for what feels like another hour. Finally, a messenger comes and hands you a small slip of paper. You open it. The message reads: "We have decided that you will be suspended without pay. When we are in need of your services again, we will contact you." It looks like for now, you are back to being a simple merchant captain.`
+			`	You are led back out to the lobby, where you wait for what feels like another hour. During that time, you speak with someone else and learn that Tomek has been removed from the Council and put under house arrest. Although interestingly, despite Tomek disobeying orders, the Senate has decided to continue to occupation of Greenrock, at least for now, their public reasoning being that it would "dishonor those who lost their lives doing what they believed to be right" to leave the planet so quickly.`
```

Comment:
```suggestion
			`	You are led back out to the lobby, where you wait for what feels like another hour. During that time, you speak with someone else and learn that Tomek has been removed from the Council and put under house arrest. Although interestingly, despite Tomek disobeying orders, the Senate has decided to continue the occupation of Greenrock, at least for now, because, “it would dishonor those who lost their lives doing what they believed to be right to leave the planet so quickly.”`
```